data/Chrisidoko.json

```diff
@@ -0,0 +1,50 @@
+{
+   "name":"Eddie Jaoude",
```

Comment:
I don't think this is correct?

Reply:
Yes i am aware i will make corrections to it

Reply:
Done